network/dime_message.ksy

```diff
@@ -0,0 +1,78 @@
+meta:
+  id: dime_message
+  title: DIME (Direct Internet Message Encapsulation) Message
+  xref:
```

Comment:
Does this make sense?

```suggestion
  xref:
    mime: application/dime
```

Reply:
I don't think they are used very often, but you are right that if they're specified we should include them.